build.gradle

```diff
@@ -40,6 +40,14 @@ allprojects {
         maven {
             url "https://download.servo.org/nightly/maven"
         }
+
+        // Application services is currently published out of nalexander's bintray for reasons.
+        // FIXME: Remove after https://github.com/mozilla/application-services/issues/252
+        // is resolved.
+        maven {
+            name 'appservices'
+            url 'https://dl.bintray.com/ncalexander/application-services'
+        }
```

Comment:
Will we synchronize the artifact from there to jcenter like the others?

Reply:
I don't fully understand the question. If I'm being honest, I was instructed to do this by @ncalexan, who hopefully does.

Reply:
It looks like we can just roll back https://github.com/mozilla-mobile/android-components/pull/1765/commits/eabc299a0f4e2bfa40093fda6c580413d9ba9dd1 'cuz `org.mozilla.appservices:rustlog` made it to jcenter: https://jcenter.bintray.com/org/mozilla/appservices/rustlog/.  I don't understand why it told me it couldn't be mirrored... maybe I did it without realizing it the first time?

Reply:
That commit is gone now, and things work. Resolving.